### DISCUSSION TITLE
LoadBalancing policy by weightage configuration

### DISCUSSION THREAD

**Discussion by ryzam** [dotnet/yarp#1187] [Q&A] [answered]
Hi guys

I wonder, is there any way to configure yarp proxy to handle loadbalancing route by weightage. Eg: If i have 2 destinations datacenter but because i have a different server spec then i need to set 60:40 request percentage.

**Comment by karelz** [comment, marked as answer]
Triage: You can implement your own LoadBalancing via `ILoadBalancingPolicy` interface. Sample: https://microsoft.github.io/reverse-proxy/articles/load-balancing.html#extensibility
You can also pass weights via [Metadata](https://microsoft.github.io/reverse-proxy/api/Yarp.ReverseProxy.Configuration.DestinationConfig.html#Yarp_ReverseProxy_Configuration_DestinationConfig_Metadata) to your LoadBalancer.

We plan to solve it in general post-1.0 -- see #393 and #395
